pkg/dockercreds/k8s_keychain_test.go

```diff
@@ -1,4 +1,4 @@
-package secret_test
+package dockercreds
```

Comment:
Are we testing anything internal in this package? if not can we change the package to `dockercreds_test`?

Reply:
We are not currently exporting the k8sGitKeychain. I would like to keep this in the dockercreds package.